ehcache-core/src/main/java/org/ehcache/core/spi/store/heap/SizeOfEngineProvider.java

```diff
@@ -37,4 +37,6 @@ public interface SizeOfEngineProvider extends Service {
    */
 
   SizeOfEngine createSizeOfEngine(ResourceUnit resourceUnit, ServiceConfiguration<?, ?>... serviceConfigs);
+
+  SizeOfEngine getNoopSizeOfEngine();
```

Comment:
What is this for?

Reply:
I have added this to cater other part of `createSizeOfEngine(-,-)` method, Now removed this, and marked the entire Interface as `@Deprecated`